internal/cloud/service.go

```diff
@@ -117,7 +117,8 @@ var ErrNotCloudManaged = errors.New("this robot is not cloud managed")
 
 // Metadata contains app-related information about the robot.
 type Metadata struct {
-	RobotPartID  string
-	PrimaryOrgID string
-	LocationID   string
+	PrimaryOrgID  string
+	LocationID    string
+	MachineID     string
+	MachinePartID string
 }
```

Comment:
note that we are removing `RobotPartID` altogether and replacing it with `MachinePartID`. this is an internal struct so no external clients should be relying on it.